mlflow/utils/search_utils.py

```diff
@@ -31,40 +39,51 @@ class SearchUtils(object):
     VALID_REGISTERED_MODEL_SEARCH_COMPARATORS = CASE_INSENSITIVE_STRING_COMPARISON_OPERATORS.union(
         {"="}
     )
-    VALID_SEARCH_ATTRIBUTE_KEYS = set(RunInfo.get_searchable_attributes())
-    VALID_ORDER_BY_ATTRIBUTE_KEYS = set(RunInfo.get_orderable_attributes())
+    VALID_SEARCH_RUN_ATTRIBUTE_KEYS = set(RunInfo.get_searchable_attributes())
```

Comment:
Can we split out run-specific constants and registry-specific constants into separate classes? perhaps `RunsSearchUtils` and `ModelRegistrySearchUtils`?